src/pages/SmartWeaveExplorer/Contract/Contract.vue

```diff
@@ -316,12 +322,8 @@
                       {{ data.item.blockHeight }}
                     </template>
 
-                    <template #cell(owner)="data">
-                      <a
-                        v-if="!isTestnet"
-                        :href="`https://v2.viewblock.io/arweave/address/${data.item.owner}`"
-                        target="_blank"
-                      >
+                    <template #cell(creator)="data">
+                      <a v-if="!isTestnet" :href="`#/app/creator/${data.item.owner}`" target="_blank">
```

Comment:
target blank does not make sens when we are linking to the subpage of the app (verify if it does not occur in other components as well)